app/views/decidim/budgets/admin/projects/show.html.erb

```diff
@@ -0,0 +1,76 @@
+<% add_decidim_meta_tags(
```

Comment:
I think this file shouldn't be in the admin folder but in app/views/decidim/budgets/projects/index.html.erb